### ISSUE TITLE
Please remove bad and unnecessary permissions

### ISSUE THREAD

**Issue by serrq** [thatsmanmeet/Tasky#84]
This app works 100% offline. No need to view network connections.

![1000100490](https://github.com/thatsmanmeet/Tasky/assets/113377431/822b41ae-c9e1-42de-85c3-54dde542d108)

**Comment by thatsmanmeet** [comment]
Wait that's a bug my app doesn't have any network permission in the android manifest. These are all the permissions that my app use and there's no mention of the view network connection permission

```xml
 <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM"/>
    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
    <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
    <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM"/>
    <uses-permission android:name="android.permission.USE_EXACT_ALARM" />
```

**Comment by thatsmanmeet** [comment]
Oh I got it, that permission might be from some library as mentioned in some of these stack overflow threads.

https://stackoverflow.com/questions/65832322/android-app-has-permission-view-network-connections-without-it-being-listed-in
https://stackoverflow.com/questions/60142230/android-app-is-able-to-make-network-calls-without-network-permissions-defined-in?rq=3
https://stackoverflow.com/questions/30660853/my-android-app-suddenly-requires-permissions-not-set-in-androidmanifest-xml-why?rq=3

Don't worry I will fix this. I got it why this is happening. Will fix that in next release
